libmisc/list.c

```diff
@@ -241,6 +241,7 @@ bool is_on_list (char *const *list, const char *member)
 
 	if ('\0' == *members) {
 		*array = (char *) 0;
+		free (members);
```

Comment:
Also free(members) at the bottom of the function?

Reply:
Also fixed